jwk.go

```diff
@@ -174,7 +174,11 @@ func (k JSONWebKey) MarshalJSON() ([]byte, error) {
 	return json.Marshal(raw)
 }
 
+var errUnsupportedJWK = errors.New("go-jose/go-jose: unsupported json web key")
```

Comment:
Hey @jsha - unless I'm missing something - seems this error is unused?

Reply:
I didn't notice this was unused during review, good eye

Reply:
I'd like to hear from @jsha in case the mistake here is that one of the code paths should return this error, but you could go ahead and open one if you'd like (otherwise, I'll do it)

Reply:
All good @mcpherrinm - I'll leave you to it.

> I'd like to hear from @jsha in case the mistake here is that one of the code paths

Yeah good call out -  I did think that might be the case/intention and thought _maybe_ it was to be tied in here? But the error message is of course slightly different in purpose.

https://github.com/go-jose/go-jose/pull/130/files#diff-adb413cfea72e48e90449f9604b1472d63b22160f9c1230bb22517ece6a0fa89R248

Reply:
Decided to cut a PR here 👍 

https://github.com/go-jose/go-jose/pull/135